src/olympia/files/tests/test_utils_.py

```diff
@@ -325,8 +325,8 @@ def test_guid(self):
                     'id': 'some-id'}}})['guid'] == 'some-id'
 
     def test_name_for_guid_if_no_id(self):
-        """Use the name for the guid if there is no id."""
-        assert self.parse({'name': 'addon-name'})['guid'] == 'addon-name'
+        """Don't use the name for the guid if there is no id."""
+        assert self.parse({'name': 'addon-name'})['guid'] is None
```

Comment:
According to the spec there's only `id` and `name` is the actual name of the add-on. So I changed it, with good intentions.